game/src/common/waypoints.rs

```diff
@@ -69,8 +66,9 @@ impl InputWaypoints {
         let mut delete_buttons = Vec::new();
 
         for (idx, waypt) in self.waypoints.iter().enumerate() {
+            let order = char::from_u32('A' as u32 + idx as u32).unwrap();
```

Comment:
Since we do this twice, how about a little helper `fn index_to_letter(usize) -> char`? That's an awful name but uh... only idea

Reply:
Updated.